Formula/sapling.rb

```diff
@@ -31,19 +31,29 @@ class Sapling < Formula
   depends_on "openssl@1.1"
   depends_on "python@3.11"
 
+  # `setuptools` 66.0.0+ only supports PEP 440 conforming version strings.
+  # Modify the version string to make `setuptools` happy.
+  # TODO: Upstream builds their own Homebrew bottles. If upstream maintainers
+  # choose to handle the version string differently, follow them:
+  #   https://github.com/facebook/sapling/tree/main/eden/scm/packaging/mac
```

Comment:
I think I'll file an upstream issue about this, since their next bottle build will almost certainly fail. That shouldn't keep us from shipping this though.

Reply:
https://github.com/facebook/sapling/issues/598